app/src/components/blocks/_delete/deleteResultStage/DeleteResultStage.tsx

```diff
@@ -1,23 +1,23 @@
 import React, { Dispatch, MouseEvent, SetStateAction, useEffect } from 'react';
 import { ButtonLink, Card } from 'nhsuk-react-components';
-import { routes } from '../../../types/generic/routes';
+import { routes } from '../../../../types/generic/routes';
 import { Link } from 'react-router-dom';
 import { useNavigate } from 'react-router';
-import useRole from '../../../helpers/hooks/useRole';
-import { REPOSITORY_ROLE } from '../../../types/generic/authRole';
-import { LG_RECORD_STAGE } from '../../../types/blocks/lloydGeorgeStages';
-import { DOWNLOAD_STAGE } from '../../../types/generic/downloadStage';
-import ReducedPatientInfo from '../../generic/reducedPatientInfo/ReducedPatientInfo';
-import { focusLayoutDiv } from '../../../helpers/utils/manageFocus';
-import useTitle from '../../../helpers/hooks/useTitle';
+import useRole from '../../../../helpers/hooks/useRole';
+import { REPOSITORY_ROLE } from '../../../../types/generic/authRole';
+import { LG_RECORD_STAGE } from '../../../../types/blocks/lloydGeorgeStages';
+import { DOWNLOAD_STAGE } from '../../../../types/generic/downloadStage';
+import ReducedPatientInfo from '../../../generic/reducedPatientInfo/ReducedPatientInfo';
+import { focusLayoutDiv } from '../../../../helpers/utils/manageFocus';
+import useTitle from '../../../../helpers/hooks/useTitle';
 
 export type Props = {
     numberOfFiles: number;
     setDownloadStage?: Dispatch<SetStateAction<DOWNLOAD_STAGE>>;
     setStage?: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
 };
 
-function DeletionConfirmationStage({ numberOfFiles, setStage, setDownloadStage }: Props) {
+function DeleteResultStage({ numberOfFiles, setStage, setDownloadStage }: Props) {
     const navigate = useNavigate();
     const role = useRole();
 
```

Comment:
the delete result page seems different from the figma design:

![image](https://github.com/nhsconnect/national-document-repository/assets/127404525/b5e5e489-be84-4c12-9ab0-c128172b5881)

Reply:
Corrected